x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -375,7 +377,7 @@ public class Constants {
         "cluster:monitor/xpack/usage/analytics",
         "cluster:monitor/xpack/usage/archive",
         "cluster:monitor/xpack/usage/ccr",
-        DataLifecycle.isEnabled() ? "cluster:monitor/xpack/usage/data_lifecycle" : null,
+        "cluster:monitor/xpack/usage/data_lifecycle",
```

Comment:
Thanks for looking into this @ywangd 

I'm not entirely sure this is the right way to fix this in 8.9 though (removing the feature flag essentially enables functionality.

I believe the test clusters should always have the feature flags enabled. So the `data_lifecycle` action should remain guarded by the feature flag and the test clusters should receive the DLM flag enabled since `8.9.0` similar to how the index mode feature flag here https://github.com/elastic/elasticsearch/blob/8.9/x-pack/plugin/security/qa/operator-privileges-tests/build.gradle#L41

Reply:
@andreidan This is not removing the feature flag. This action [was moved](https://github.com/elastic/elasticsearch/pull/97442/files#diff-95b3a7296287318fc681b3c464dd8ddc7042e05be37e638c17d0ff454d137f1eL62) out of the feature flag protection in 8.9. This change is merely reflecting that change in test code. In another word, there is no change to production behaviour.

Reply:
Ah ++ Good catch